layer/common_logging.cc

```diff
@@ -62,14 +61,13 @@ std::string EventToCommonLogStr(Event &event) {
         break;
       }
     }
-    if (i + 1 != e) csv_str << ",";
   }
   return csv_str.str();
 }
 
 CommonLogger::CommonLogger(const char *filename) {
   if (filename) {
-    out_ = fopen(filename, "w");
+    out_ = fopen(filename, "a");
```

Comment:
Can you add a short comment explaining why we want to append? (IUUC, because multiple layers will write to the same file at the same time)